webui/react/src/pages/DesignKit.tsx

```diff
@@ -606,8 +606,7 @@ const ChartsSection: React.FC = () => {
       [XAxisDomain.Batches]: line1BatchesDataStreamed,
       [XAxisDomain.Time]: [],
     },
-    metricType: MetricType.Training,
-    name: 'Line',
+    name: 'training.Line',
```

Comment:
question: why is it hardcoded instead of `MetricType.Training`?

Reply:
we're removing the `MetricType` type so that all metric-groups are equal, and `LineChart` can be used by other groups within HPE without our own logic